Sources/Rendering/Core/ColorTransferFunction/index.d.ts

```diff
@@ -1,10 +1,10 @@
 import vtkDataArray from '../../../Common/Core/DataArray';
-import { vtkObject } from '../../../interfaces';
+import vtkScalarsToColors from '../../../Common/Core/ScalarsToColors'
 import { ColorSpace, Scale } from "./Constants";
 
 
 /* TODO: use VtkScalarsToColors instead of VtkObject */
-export interface vtkColorTransferFunction extends vtkObject {
+export interface vtkColorTransferFunction extends vtkScalarsToColors {
```

Comment:
remove the TODO?

Reply:
oops, how did I missed that? just fixed it